content/pytorch/concepts/tensor-operations/terms/t/t.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: '.t()'
+Description: 'Returns the transpose of a given 2D tensor.'
+Subjects:
+  - 'AI'
+  - 'Data Science'
+Tags:
+  - 'AI'
+  - 'Arrays'
+  - 'Data Structures'
+  - 'Deep Learning'
+CatalogContent:
+  - 'intro-to-py-torch-and-neural-networks'
+  - 'paths/computer-science'
+---
+
+In PyTorch, the **`.t()`** method returns the transpose of a given 2D tensor. If the tensor is a 0D or 1D tensor, the method returns it as it is.
+
+## Syntax
+
+```pseudo
+torch.t(ten)
```

Comment:
```suggestion
torch.t(ten)
```
In PyTorch , the .t( ) is an instance method so it should be called directly on the object like this `ten.t() `. Don't use torch,t ( ), because it does not exist as a function.

Reply:
Hey @SaviDahegaonkar, I found out that both `torch.t()` and `ten.t()` can be used to call the `.t()` method.

Also, the official documentation for [`ten.t()`](https://pytorch.org/docs/stable/generated/torch.Tensor.t.html#torch.Tensor.t) redirects us to [`torch.t()`](https://pytorch.org/docs/stable/generated/torch.t.html#torch.t).

So, I think we can go with `torch.t()`. What do you think?

Reply:
Yeah, in that case, I must approve this PR.